common/beerocks/bcl/source/son/son_wireless_utils.cpp

```diff
@@ -557,6 +559,82 @@ std::vector<uint8_t> wireless_utils::calc_5g_20MHz_subband_channels(
     return channels;
 }
 
+void wireless_utils::get_channel_preferences(
+    beerocks::message::sWifiChannel supported_channels[],
+    std::map<beerocks::message::sPreference, std::list<beerocks::message::sWifiChannel>>
+        &preferences)
+{
+    std::list<uint8_t> oper_classes;
```

Comment:
For all these things, I actually think a std::vector makes more sense than a std::list. In theory std::list is more efficient for appending, but that's only due to resize operations. The cost, however, is 2 pointers overhead per element. If we presize the vector to e.g. 8 elements (which makes it about as large as a single-entry std::list), it will almost never be resized.

Anyway, that's just an optimisation so not very important.

Reply:
Done.